app/Util.js

```diff
@@ -13,8 +13,25 @@ export function nowStr() {
   return dayjs().format('H:mm');
 }
 
+export function showAlert(title, description, onOkPress, okText, onCancelPress, cancelText) {
```

Comment:
I'm not sure I understand the purpose of this wrapper. It seems to do the same thing as the Alert native module.

Reply:
@efalkner To reduce code redundancy, every time someone write the full alert code to show one

Reply:
I like the thought of abstraction in general, but I'm not sure if this particular one is providing much value. It looks as though this hasn't really abstracted much away from the consumer, and could potentially introduce bugs, for example, if they don't pass in arguments in the right order.